### PR TITLE
Enhancement: replaced nanojpeg with libjpeg-turbo library

### PR DESCRIPTION
**Pull request by NightlyFox**
it was only one function, so i went ahead and took the liberty of doing this myself! 👍 i tested it, and all the icons seem to be loading correctly!

### CODE REVIEW

File: common/menu-list.c [hunk 1]
```diff
@@ -209,10 +209,15 @@ int themeMenuScan(const char* target) {
     menuEntry_s* me = menuCreateEntry(ENTRY_TYPE_THEME);
 
     if(me) {
+<<<<<<< HEAD
```
Comment (fincs) [inline common/menu-list.c]: Merge conflicts...

Comment (NightlyFox) [reply common/menu-list.c]: I resolved them on vs code... but for some reason  it didnt commit the first time... i had to add a newline and remove some spaces and then recommit for it to reflect the changed

**Comment by yellows8** [comment]
rebase

**Comment by NightlyFox** [comment]
> rebase

Never done this beforr 🤔 how do i do this?

**Comment by NightlyFox** [comment]
i might have to cancel this pull request and then try it again with a new branch

**Pull request closed without merging**